SpatialGDK/Source/SpatialGDKTests/Private/RPCContainer/RPCContainerTest.cpp

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestDefinitions.h"
+
+#include "ObjectDummy.h"
+#include "ObjectSpy.h"
+
+#include "Utils/RPCContainer.h"
+#include "Schema/RPCPayload.h"
+
+#include <Core.h>
+
+#define RPCCONTAINER_TEST(TestName) \
+	TEST(FRPCContainer, TestName)
+
+using namespace SpatialGDK;
+
+namespace
+{
+	ESchemaComponentType AnySchemaComponentType = ESchemaComponentType::SCHEMA_ClientReliableRPC;
+	ESchemaComponentType AnyOtherSchemaComponentType = ESchemaComponentType::SCHEMA_ClientUnreliableRPC;
+
+	FUnrealObjectRef GenerateObjectRef(UObject* TargetObject)
+	{
+		return FUnrealObjectRef{ Worker_EntityId(TargetObject), 0 };
+	}
+
+	uint32 GeneratePayloadFunctionIndex()
+	{
+		static uint32 FreeIndex = 0;
+		return FreeIndex++;
+	}
+
+	FPendingRPCParamsPtr CreateMockParameters(UObject* TargetObject, ESchemaComponentType Type)
+	{
+		// Use PayloadData as a place to store RPC type
+		RPCPayload Payload(0, GeneratePayloadFunctionIndex(), SpyUtils::TypeToArray(Type));
+		int ReliableRPCIndex = 0;
+
+		FUnrealObjectRef ObjectRef = GenerateObjectRef(TargetObject);
+
+		return MakeUnique<FPendingRPCParams>(ObjectRef, MoveTemp(Payload), ReliableRPCIndex);
+	}
+}
```

Comment:
`  // anonymous namespace`

Reply:
Also github why are you trying to change the spacing of my code snippets?

Reply:
Added.